manim/utils/tex_file_writing.py

```diff
@@ -205,4 +205,14 @@ def convert_to_svg(dvi_file, extension, page=1):
             os.devnull,
         ]
         os.system(" ".join(commands))
+
+    # if the file does not exist now, this means conversion failed
+    if not os.path.exists(result):
+        raise ValueError(
+            f"Your installation does not support converting {extension} files to SVG."
+            f" Consider updating dvisvgm to at least version 2.4."
+            f" If this does not solve the problem, please refer to our troubleshooting guide at:"
+            f" https://manimce.readthedocs.io/en/latest/installation/troubleshooting.html"
```

Comment:
Pro tip: add a label to the heading within the RST document so that you can link directly to the corresponding heading and not the whole troubleshooting page (which will, in the future, grow in size).

Reply:
Done. Hope I got the syntax right.